### PR TITLE
Make two function-static variables const

### PR DESCRIPTION
**Pull request by makortel**
#### PR description:

This PR makes two function-static variables const to silence static analyzer warnings (that caught my eye in the tests of another PR). In one of them I also simplified the static variable to be `std::array` instead of `std::vector`.

#### PR validation:

Code compiles

### CODE REVIEW

**Comment by makortel** [comment]
@cmsbuild, please test

File: FWCore/Framework/src/ExceptionActions.cc [hunk 1]
```diff
@@ -8,23 +8,16 @@
...
-      static ActionNames tab;
-      return static_cast<unsigned int>(code) < tab.table_.size() ? tab.table_[code] : "UnknownAction";
+      static const std::array<char const*, LastCode> tab = []() {
+        std::array<char const*, LastCode> table;
+        table[IgnoreCompletely] = "IgnoreCompletely";
+        table[Rethrow] = "Rethrow";
+        table[SkipEvent] = "SkipEvent";
+        table[FailPath] = "FailPath";
+        return table;
+      }();
```
Comment (makortel) [inline FWCore/Framework/src/ExceptionActions.cc:19]: @Dr15Jones What do you think about this change?

Comment (Dr15Jones) [reply FWCore/Framework/src/ExceptionActions.cc:19]: Could it be constexpr?

Comment (makortel) [reply FWCore/Framework/src/ExceptionActions.cc]: Done. I want to note that I was tempted to use the initializer list to initialize the array, but I valued more the consistency of taking the indices of elements from the enum.

**Comment by makortel** [comment]
@cms-sw/externals-l2 Did the tests get stuck?

**Comment by iarspider** [comment]
@makortel the comparision job timed out waiting for CVMFS deployment. I have restarted it.

**Comment by makortel** [comment]
+1

**Comment by perrotta** [comment]
+1

**Pull request merged**